i18n/en.json

```diff
@@ -3504,22 +3504,6 @@
     "translation": "Page not found"
   },
   {
-    "id": "leave_team_modal.desc",
```

Comment:
Those are client strings. I'm not sure why they were even on the server to begin with so I just removed them.